assets/js/modules/reader-revenue-manager/components/settings/SettingsEdit.js

```diff
@@ -44,13 +44,19 @@ import SettingsNotice, {
 	TYPE_INFO,
 } from '../../../../components/SettingsNotice';
 import WarningIcon from '../../../../../../assets/svg/icons/warning-icon.svg';
+import StoreErrorNotices from '../../../../components/StoreErrorNotices';
 
 export default function SettingsEdit() {
 	const isRRMv2Enabled = useFeature( 'rrmModuleV2' );
 
+	const productIDs = useSelect( ( select ) => {
```

Comment:
Hey @juniovitorino, don't forget to remove this `productIDs` declaration as it's no longer needed :)